todo-backend/README.md

```
@@ -0,0 +1,274 @@
+- **Params**  
+  None
+- **Query**
+
+  - Optional: `page=[integer]` (Page number for pagination, defaults to 1)
+  - Optional: `limit=[integer]` (Number of items per page)
+
+- **Success Response:**
+
+  - **Code:** 200
+  - **Content:**
+    ```json
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should be surrounded by blank lines
null

(MD031, blanks-around-fences)